scalableminds/webknossos
# Add details view for publications

This PR adds a view for each publication at `/publication<publication_id>`. This page can be accessed via a link in the spotlight view's publication cards. The link is next to the title of the publication and is only visible when the title is hovered.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Enable the fontendpage in the features (`appplication.conf`) and just open the spotlight view.
- From there select on of the publications link to its own view.
- From the "detailed" view you can go back by using the icon next to "Back".  
- This should work for each publication.

### Issues:
- fixes #3904 

------
